### PR TITLE
Delta-V Specific Posters!

### PR DESCRIPTION
**Pull request by IAmNotGray**
## About the PR
Adds seven new posters for mappers to use in their maps.
## Why / Balance
I'd like to see more Delta-specific posters. Also wanted to make posters for the game, anyway... Credits to JustAnOrange for making some of the posters featured in this PR!

## Technical details
Posters + meta files and the like
Hassled for hours trying to get it to work.
Random Spawns included

## Media
dangernana
getwork
syndicateandyou
nanotrasenandyou
nanotrasenwork
syndicatework
l6poster

## Requirements

- [x] I have tested all added content and changes.
- [x] I have added media to this PR or it does not require an ingame showcase.

## Breaking changes
Posters..?-

**Changelog**

:cl: This_Is_Gray & JustAnOrange added:
- add: Seven new posters have arrived to NanoTrasen stations.

### CODE REVIEW

**Comment by Kr8art** [comment]
Very cool! Can't wait to slip on a banana peel while the nearby clown gleefully spam-points at first poster.

File: Resources/Prototypes/DeltaV/Entities/Structures/Wallmounts/Signs/posters.yml [hunk 1]
```diff
@@ -27,3 +27,73 @@
...
+  components:
+  - type: Sprite
+    sprite: DeltaV/Structures/Wallmounts/Posters/grayposters.rsi
+    state: work
+    
+- type: entity
+  parent: PosterBase
+  id: PosterContrabandL6
+  name: "LITDAKKA"
+  description: "The notorious L6 light machine gun. Produced by Donk Co., This monster of a weapon is capable of cutting down dozens of people in mere moments. And to think this company also makes party supplies!"
```
Comment (Unkn0wnGh0st333) [inline Resources/Prototypes/DeltaV/Entities/Structures/Wallmounts/Signs/posters.yml]:
```suggestion
  description: "The notorious L6 light machine gun. Produced by Waffle Co., This monster of a weapon is capable of cutting down dozens of people in mere moments. And to think this company also makes party supplies!"
```

Comment (IAmNotGray) [reply Resources/Prototypes/DeltaV/Entities/Structures/Wallmounts/Signs/posters.yml]: Insert Nerd Emoji Here

Comment (Unkn0wnGh0st333) [reply Resources/Prototypes/DeltaV/Entities/Structures/Wallmounts/Signs/posters.yml]: :trollface:

File: Resources/Prototypes/DeltaV/Entities/Structures/Wallmounts/Signs/posters.yml [hunk 2]
```diff
@@ -27,3 +27,73 @@
...
+    
+- type: entity
+  parent: PosterBase
+  id: PosterLegitBananaWarning
+  name: "The Dangers of the Peel"
+  description: "A poster warning the viewer of the dangers of discarded banana peels. Injuries can include, but are limited to, blunt force trauma to the head, skull fractures, and broken bones."
+  components:
+  - type: Sprite
+    sprite: DeltaV/Structures/Wallmounts/Posters/grayposters.rsi
+    state: dangernana
```
Comment (deltanedas) [inline Resources/Prototypes/DeltaV/Entities/Structures/Wallmounts/Signs/posters.yml]: add newline to end of file

File: Resources/Textures/DeltaV/Structures/Wallmounts/Posters/grayposters.rsi/meta.json [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    "copyright": "Original work by Gray & JustAnOrange.",
+    "size": {
+        "x": 32,
+        "y": 32
+    },
+    "states": [
+        {
+            "name": "posterearthnanotrasen"
+        },
+		{
```
Comment (deltanedas) [inline Resources/Textures/DeltaV/Structures/Wallmounts/Posters/grayposters.rsi/meta.json]: indents fucked

**Review by deltanedas — CHANGES_REQUESTED** [review]
add these to legit and contraband spawners

**Comment by IAmNotGray** [comment]
I forgot to remove the test.rsi folder before making the PR. I'm so good at this.

**Comment by deltanedas** [comment]
almost every line of that file has mixed tabs/spaces

**Comment by IAmNotGray** [comment]
> almost every line of that file has mixed tabs/spaces

You can tell how good at this I am.

File: Resources/Textures/DeltaV/Structures/Wallmounts/Posters/grayposters.rsi/meta.json [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+		"x": 32,
+		"y": 32
+	},
+	"states": [
+		{
+			"name": "posterearthnanotrasen"
+		},
+		{
+			"name": "posterroidsyndicate"
+        },
```
Comment (deltanedas) [inline Resources/Textures/DeltaV/Structures/Wallmounts/Posters/grayposters.rsi/meta.json:12]: last one

File: Resources/Textures/DeltaV/Structures/Wallmounts/Posters/grayposters.rsi/meta.json [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+		},
+		{
+			"name": "litdakka"
+		},
+		{
+			"name": "dangernana"
+		}
+	
+	]
+}
```
Comment (deltanedas) [inline Resources/Textures/DeltaV/Structures/Wallmounts/Posters/grayposters.rsi/meta.json]: newline at end of file

**Review by deltanedas — CHANGES_REQUESTED** [review]

**Comment by IAmNotGray** [comment]
Okay hopefully that's all I need to fix-
I am NOT a coding type of guy.

**Comment by IAmNotGray** [comment]
Anything on this PR?

**Comment by Monotheonist** [comment]
These look banger, and I think they're cool! And all tests have passed so all it needs is someone to look at it and say yes, pretty much.

File: Resources/Textures/DeltaV/Structures/Wallmounts/Posters/grayposters.rsi/meta.json [hunk 6]
```diff
@@ -0,0 +1,32 @@
+{
```
Comment (deltanedas) [inline Resources/Textures/DeltaV/Structures/Wallmounts/Posters/grayposters.rsi/meta.json:1]: this file is still using tab indents instead of 4 spaces

**Review by deltanedas — CHANGES_REQUESTED** [review]

**Review by deltanedas — APPROVED** [review]

**Pull request merged**